### PR TITLE
Feature/styles skeleton

### PR DESCRIPTION
**Pull request by battan40**
#### Is this a fix or a feature?

* This is a feature

#### What is the change/fix?

* Adds the basics of css styling

#### Where should the reviewer start?

* html, css files
* verify both presence and accuracy of rules and corresponding tags
* run open src/index.html verify consistency across design

#### Any background information you want to provide:

i have an image disappearing, please look at html tag:
`<section class="sign-square-info"></section>  `

* it is responsive to design but on the full size screen this tag disappears

### CODE REVIEW

**Review by EllieJellyBean — COMMENTED** [review]
This all looks really good! Everything is well organized and I absolutely love your theme! I didn't see anything concerning or that stood out as a problem - Good job!

File: src/styles.css [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  grid-gap: 1%;
+  padding: 2%;
+  box-shadow:
+  0px 0px 2px 2px rgba(0, 0, 0, 0.5),
+  0px 0px 4px 4px rgba(0, 0, 0, 0.3)
+}
+
+.sign-square {
+  position: relative;
+}
```
Comment (npdarrington) [inline src/styles.css:88]: On your grid, the pictures get stretched out of proportion if the screen is really large. For your articles that are making your cards, think about setting a min-width and max-width. This way they don't compress too tiny on small devices and stretch too far on large devices.

File: src/styles.css [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  .recipient-wrapper {
+    height: 400vh;
+    width: 100%;
+  }
+
+  .recipient-grid,
+  .sign-square {
+    display: grid;
+    grid-template-columns: repeat(2, auto);
+    box-shadow: none;
```
Comment (npdarrington) [inline src/styles.css:124]: Emulating a phone screen inside of Google Chrome dev tools, something like an iPhone, the articles are looking compressed and out of proportion. Add one more breakpoint that is aimed at a cell phone or smaller mobile devices that take the grid template down to a single column.

File: src/index.html [hunk 3]
```diff
@@ -3,61 +3,202 @@
   <head>
     <meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Static Composition</title>
-    <link rel="stylesheet" href="css/normalize.min.css"
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" crossorigin="anonymous" />
+    <link rel="stylesheet" href="./normalize.css"
```
Comment (npdarrington) [inline src/index.html:7]: Great use of normalize! It really helps not having to always fight against the user agent stylesheet for individual browsers and helps the design remain mostly consistent across them all.

File: src/styles.css [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+  display: flex;
+  width: 100%;
+  height: 50%;
+}
+
+.sign-square-info {
+  display: flex;
+  width: 100%;
+  height: 50vh;
+  background-image: url('../assets/recipient-info.png');
```
Comment (npdarrington) [inline src/styles.css:108]: Take a 2nd look at this background image for your sign-square-info. For some reason, on my screen, it is not showing the information on the bottom on larger screens, however, when it hits the media query, it shows for smaller devices. Is this only happening on my screen?

Screen Shot 2021-04-11 at 3 33 32 PM

Screen Shot 2021-04-11 at 3 33 57 PM

**Review by npdarrington — APPROVED** [review]
This is looking really solid! The theme is great! A hover effect on your arrows to show they can be clicked on will go a long way.

File: src/styles.css (hunk 4 above)
Comment (battan40) [reply src/styles.css:108]: Its definitely happening on my screen too.  I will pull this back down to attempt a resolution.

**Pull request merged**